src/Modules/Authontication/Components/LogIn/LogIn.tsx

```diff
@@ -1,7 +1,144 @@
-import React from 'react'
+import { useForm } from 'react-hook-form';
+
+import { useContext, useState } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import axios from 'axios';
+import { toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import { USERS_URLs } from '../../../../constans/END_POINTS';
+import { AuthContext } from '../Context/AuthContext'
+import { EmailValidation, PasswordValidation } from '../../../../constans/VALIDATIONS';
 
 export default function LogIn() {
+  let { saveLoginData } = useContext(AuthContext);
+  let navigate = useNavigate();
+  let{
+    register,
+    handleSubmit,
+    formState:{errors},
+  } = useForm({ mode: "onBlur" });
+
+
+  const [isPasswordVisible, setIsPasswordVisible] = useState(false);
+  const toggleVisibility = (setterFunction: any) => {
+    setterFunction((prevState: any) => !prevState);
+  };
+
+  let onSubmit = async (data:any)=>{
+    try {
+      let response = await axios.post(USERS_URLs.Login, data);
+      localStorage.setItem('token', response.data.token);
+      saveLoginData();
+      console.log(response)
+      navigate('/dashboard');
+      toast.success(
+        response.data.message || 'congratulations, login success !'
+      );
+    
+    console.log(response)
+      } 
+     catch (error:any) {
+     toast.error(
+      error?.response?.data?.message || "Login unsuccessful. Please try again"
+    );
+     console.log(error);
+    }
+  }
   return (
-    <h1 className='text-white'>LogIn</h1>
+    <>
+    <div className="auth-title my-4">
+    <p className="text-white">Welcome to PMS</p>
+    <h3 className="main-color title">
+          <span className="frist-ch position-relative">L</span>ogin
+        </h3>
+  </div>
+
+  <form className='container-form' onSubmit={handleSubmit(onSubmit)}>
+    <div className=" my-4">
+    <label className="main-color my-1">Email</label>
+      <div className="input-group ">
+      <input type="text" className="form-control form-input"
+       placeholder="Enter your email"
+       aria-label="email" aria-describedby="basic-addon1"
+       {...register("email",EmailValidation)}
+       />
+      </div>
+     
+    </div>
+    {errors.email && (
+            <span className="text-danger">
+              {String(errors.email.message)}
+            </span>
+    )}
+    
+       
+    <div className="my-4">  
+      <label className="main-color my-1">Password</label>
+      <div className='input-group'>
+      <input type={`${isPasswordVisible?"text" : "password"  }`}
+       className="form-control form-input" 
+       placeholder="Enter your password"
+       aria-label="password"
+       aria-describedby="basic-addon1"
+       {...register("password", PasswordValidation)}/>
```

Comment:
It should only be required, we shouldn't add password validation here as this is done when creating or changing the password. 
Adding the validation in the login will prevent people who signed up before the validation is added or after it has been changed from being able to login as they should match the new or the updated validation from the backend.